srcs/HandlingString/HandlingString.cpp

```diff
@@ -0,0 +1,325 @@
+}
+
+bool	HandlingString::check_lastword_semicoron(std::string const &word)
+{
+	size_t	pos = 0;
+	size_t	semicoron_count = 0;
+
+	while (word[pos] != '\0')
+	{
+		if (word[pos] == ';')
+			semicoron_count++;
+		pos++;
+	}
+	if (semicoron_count != 1)
+		return (false);
+	if (word[pos - 1] != ';')
+		return (false);
+	return (true);
+}
+
+std::string	HandlingString::skip_lastsemicoron(std::string const &word)
+{
+	size_t		pos = 0;
+	std::string	return_str;
+	size_t		index = word.find(';');
+
+	if (index == std::string::npos)
+		return (word);
+	while (word[pos] != ';')
+	{
+		return_str = return_str + word[pos];
+		pos++;
+	}
+	return (return_str);
+}
+
+void	HandlingString::error_show(std::string const &word, size_t const &pos)
+{
+	std::string error_one = "<< missed word! >>";
+	std::string error_two = "<< missed line >>";
+	std::string error_three = "===============";
+
+	std::cout << error_three << std::endl;
+	std::cout << error_one << std::endl;
+	std::cout << "* " << word << std::endl;
+	std::cout << error_two << std::endl;
+	std::cout << "line > " << pos << std::endl;
+	std::cout << error_three << std::endl;
+}
+
+bool	HandlingString::compare_word(std::string const &tgt_string, std::string const &key)
+{
+	if (key == tgt_string)
+		return (true);
+	return (false);
+}
+
+std::string HandlingString::obtain_second_word(std::string const &line)
+{
+	std::istringstream	splited_woeds(line);
+	std::string			f_word;
+	std::string			s_word;
+	std::string			t_word;
+
+	splited_woeds >> f_word >> s_word >> t_word;
+	return (s_word);
+}
+
+std::string HandlingString::int_to_str(size_t pos)
+{
+	std::string result;
+
+    if (pos == 0)
+	{
+        result = "0";
+		return result;
+	}
+	while (pos > 0)
+	{
+		result =  static_cast<char>('0' + pos % 10) + result;
+		pos /= 10;
+	}
+    return result;
+}
+
+void HandlingString::show_vector_contents(std::vector<std::string> subject)
+{
+	if (subject.empty() == true)
+	{
+		std::cout << " * NO CONTENTS INSERTED * ";
+		return;
+	}
+	std::vector<std::string>::iterator it = subject.begin();
+	if (it == subject.end())
+		std::cout << *it << std::endl;
+	while (it != subject.end())
+	{
+		std::cout << *it << " ";
+		it++;
+	}
+}
+
+void HandlingString::ft_strcpy(char *input_memory, std::string const &sub)
+{
+	size_t	error_message_len = 0;
+	while (error_message_len != sub.length())
+	{
+		input_memory[error_message_len] = sub[error_message_len];
+		error_message_len++;
+	}
+	input_memory[error_message_len] = '\0';
+}
+
+std::string HandlingString::obtain_beforeword(const std::string other, char delimiter)
+{
+	return other.substr(0, other.find(delimiter));
+}
+
+std::string HandlingString::obtain_afterword(const std::string other, char delimiter)
+{
+	return other.substr(other.find(delimiter) + 1);
+}
+
+std::string	HandlingString::obtain_weight(const std::string &other)
+{
+	return (HandlingString::obtain_afterword(other, '='));
+}
+
+bool	HandlingString::check_int_or_not(const std::string &value)
```

Comment:
```suggestion
bool	HandlingString::is_int_or_not(const std::string &value)
```
IMO:
* 判定系の関数なので、`is_`が良さそうです

Q:
* int or not よりは digit or not を見ている？
* +/-などの符号はチェックしなくても大丈夫な関数ですか？
* この関数はintより大きな数値でもintと判定しますが、大丈夫でしょうか？
* int判定ならOF等のエラーハンドリング機能を持ったstr_to_intを呼び確認してもよらそうですね

Reply:
is_int_max　を使うことで後半の方は解消しました！requestで使用するので＋ーは使えないという前提です